lib/apiservers/engine/backends/backends.go

```diff
@@ -310,3 +313,23 @@ func setPortMapping(info *models.ContainerInfo, backend *Container, container *c
 	}
 	return nil
 }
+
+func loadRegistryCACerts() {
+	if len(vchConfig.RegistryCertificateAuthorities) == 0 {
+		return
+	}
+
+	rootCertPool, err := x509.SystemCertPool()
+	log.Debugf("Loaded %d CAs for registries from system CA bundle", len(rootCertPool.Subjects()))
+	if err != nil {
+		log.Fatalf("Unable to load system CAs")
```

Comment:
This will cause the personality to exit, then be restarted immediately by vic-init, repeated ad infinitum.

Use x509.NewPool() to initialize an empty pool and append to that - it's better than nothing.